js/ui/components/clubhouse.js

```diff
@@ -381,5 +395,15 @@ var ClubhouseComponent = new Lang.Class({
         if (this.proxy.g_name_owner && this._clubhouseSource)
             this._clubhouseSource.activateAction('stop-quest', null);
     },
+
+    _onPropertiesChanged: function(proxy, changedProps, invalidatedProps) {
```

Comment:
See my comments in the Clubhouse's PR, I think we should not use a property for this.